### PR TITLE
activate quotas for certificate requests per day and issuer

### PR DESCRIPTION
**Pull request by MartinWeindel**
**How to categorize this PR?**

/area control-plane
/kind task
/priority normal

**What this PR does / why we need it**:
The command line flag `--issuer.default-requests-per-day-quota` is now set with the value `100` for the deployment of the `cert-controller-manager`.
This means, the maximum number of certificate requests that can be requested on a shoot cluster for a given issuer is restricted to 100 per day. The `cert-controller-manager` currently allows a burst of 25%, this means initially or after 6h without certificate requests it will be possible to perform 25 requests at once. 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```noteworthy operator
activate quotas for certificate requests per day and issuer
```

### CODE REVIEW

File: charts/gardener-extension-shoot-cert-service/templates/_helpers.tpl [hunk 1]
```diff
@@ -8,6 +8,7 @@ apiVersion: shoot-cert-service.extensions.config.gardener.cloud/v1alpha1
 kind: Configuration
 issuerName: {{ required ".Values.certificateConfig.defaultIssuer.name is required" .Values.certificateConfig.defaultIssuer.name }}
 restrictIssuer: {{ required ".Values.certificateConfig.defaultIssuer.restricted is required" .Values.certificateConfig.defaultIssuer.restricted }}
+defaultRequestsPerDayQuota: {{ required ".Values.certificateConfig.defaultRequestsPerDayQuota is required" .Values.certificateConfig.defaultRequestsPerDayQuota }}
```
Comment (timuthy) [inline charts/gardener-extension-shoot-cert-service/templates/_helpers.tpl:11]: For backwards compatibility can you make this optional?

File: charts/gardener-extension-shoot-cert-service/values.yaml [hunk 2]
```diff
@@ -27,6 +27,7 @@ controllers:
     concurrentSyncs: 5
 
 certificateConfig:
+  defaultRequestsPerDayQuota: 100
```
Comment (timuthy) [inline charts/gardener-extension-shoot-cert-service/values.yaml:30]: For backwards compatibility can you remove or comment this line?

**Review by timuthy — COMMENTED** [review]
Looks fine, just two suggestions.

File: charts/gardener-extension-shoot-cert-service/templates/_helpers.tpl (hunk 1 above)
Comment (MartinWeindel) [reply charts/gardener-extension-shoot-cert-service/templates/_helpers.tpl]: done

File: charts/gardener-extension-shoot-cert-service/values.yaml (hunk 2 above)
Comment (MartinWeindel) [reply charts/gardener-extension-shoot-cert-service/values.yaml]: done

**Review by timuthy — APPROVED** [review]
/lgtm

**Pull request merged**